src/core/js/models/adaptModel.js

```diff
@@ -55,7 +61,7 @@ define([
 
       this.init();
 
-      _.defer(function() {
+      _.defer(()=>{
```

Comment:
Style suggestion of spaces between arrows **throughout**:
```suggestion
      _.defer(() => {
```